packages/instanceserver/src/channels.ts

```diff
@@ -253,6 +254,12 @@ const loadEngine = async (app: Application, sceneId: string) => {
     const sceneResultPromise = app.service('scene').get({ projectName, sceneName, metadataOnly: false }, null!)
 
     await initializeCoreSystems()
+
+    initSystemSync(world, {
```

Comment:
remove this, initialize in initializeCoreSystems

Reply:
Done.